app/assets/javascripts/angular/controllers/drivers_controller.js

```diff
@@ -0,0 +1,56 @@
+(function() {
+    angular
+    .module('taxi')
+    .controller('DriversController', ['$scope', '$http', function($scope, $http) {
+
+        var dispatcher = new WebSocketRails(window.location.host + '/websocket');
+        var notification = document.getElementById('notification');
```

Comment:
Avoid working with DOM from controller. It responsibility of view layer of the application. Write a directive that will handle this